init-tools.cmd

```diff
@@ -59,6 +59,12 @@ if exist "%DOTNET_CMD%" goto :afterdotnetrestore
 
 if NOT exist "%DOTNET_PATH%" mkdir "%DOTNET_PATH%"
 
+:: set registry to take dump automatically when test process crashes
+powershell -Command "New-Item -Path 'HKLM:\SOFTWARE\Microsoft\Windows\Windows Error Reporting' -Name 'LocalDumps' -Force"
+powershell -Command "New-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps' -Name 'DumpFolder' -Value '%~dp0CrashDumps' -PropertyType 'ExpandString' -Force"
+:: take full dump
+powershell -Command "New-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps' -Name 'DumpType' -Value 2 -PropertyType DWord -Force"
```

Comment:
❔ Normally these registry entries are added for one or more specific process names. Since it's on CI, it seems reasonable to not restrict it in this manner. However, are we sure that the entries will work when used without specified a process name?

Reply:
Yes, this works on my test in both local and CI

---
In reply to: [367624922](https://github.com/dotnet/machinelearning/pull/4666#discussion_r367624922) [](ancestors = 367624922)